other/install_2-1_mysql.sql

```diff
@@ -1179,9 +1179,9 @@ CREATE TABLE {$db_prefix}user_likes (
 CREATE TABLE {$db_prefix}mentions (
   content_id INT DEFAULT '0',
   content_type VARCHAR(10) DEFAULT '',
-  id_mentioned INT DEFAULT 0,
-  id_member INT(10) UNSIGNED NOT NULL DEFAULT 0,
-  `time` INT NOT NULL DEFAULT 0,
+  id_mentioned INT DEFAULT '0',
```

Comment:
it's sql standard not to use single quote für integer,
but the conversion from string to int is not so hard,
that is reason why this is working...

but this keep wrong from standard pov.

so in my eyes keep the right thing right and wrong think wrong,
or fix the wrong ones, but don't make the right one to wrong one only because it's looks better.